CHANGELOG.rst

```diff
@@ -3,14 +3,19 @@ Change Log
 
 ..
    All enhancements and patches to edx_toggles will be documented
-   in this file.  It adheres to the structure of http://keepachangelog.com/ ,
+   in this file.  It adheres to the structure of https://keepachangelog.com/ ,
    but in reStructuredText instead of Markdown (for ease of incorporation into
    Sphinx documentation and the PyPI description).
    
-   This project adheres to Semantic Versioning (http://semver.org/).
+   This project adheres to Semantic Versioning (https://semver.org/).
 
 .. There should always be an "Unreleased" section for changes pending release.
 
+Unreleased
```

Comment:
In fact... has this *ever* been released? The bottom of the changelog says "First release on PyPI" but I think that's not true—I don't see anything on PyPI for this library, and there's no deploy password set in the `.travis.yml`.

Reply:
This is not set up with PYPI, I think there is a ticket somewhere to do that